doc/classes/Node.xml

```diff
@@ -205,7 +205,7 @@
 		<method name="create_tween">
 			<return type="Tween" />
 			<description>
-				Creates a new [Tween] and binds it to this node. Fails if the node is not inside the tree.
+				Creates a new [Tween] and binds it to this node.
 				This is the equivalent of doing:
```

Comment:
It should be obvious, but might want to clarify here
```suggestion
				This is the equivalent of doing (if inside the tree):
```
Or similar

Reply:
Not sure about that. If I add this, it would be worth to mention that the method also works outside the tree, adding unnecessary details. Previously the description just stated the method does not work outside scene tree, now it always works (unless you use custom MainLoop), so I just removed the note. No people complained about that limitation, so if someone assumes from the example that it's impossible to call the method outside the tree, it's not really a problem.